pom.xml

```diff
@@ -122,8 +122,7 @@
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <!-- Release 21.0 requires JDK 1.8 or newer. -->
-        <version>20.0</version>
+        <version>26.0-jre</version>
```

Comment:
I'd just remove the version here and let it defined once in the dependencyManagement in the parent POM

Reply:
I thought this was the parent pom?

Reply:
You are right